src/django/api/models.py

```diff
@@ -825,3 +825,32 @@ class Meta:
 
     def __str__(self):
         return '{} -> {}'.format(self.oar_id, self.facility)
+
+
+class RequestLog(models.Model):
+    """
+    Logs non-webapp API requests by User
+    """
+    user = models.ForeignKey(
+        'User',
+        null=False,
+        on_delete=models.CASCADE,
+        help_text='The User account that made the request'
+    )
+    token = models.CharField(
+        max_length=40,
+        null=False,
+        db_index=True,
+        help_text='The API token used to make the request'
+    )
+    path = models.CharField(
+        max_length=2083,
+        null=False,
+        help_text='The requested resource path'
+    )
+    response_code = models.IntegerField(
+        null=True,
+        help_text='The HTTP status code returned to the requester'
+    )
+    created_at = models.DateTimeField(auto_now_add=True, db_index=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
Can't imagine this field being used very much in a table like this, but I suppose it doesn't hurt to have it for consistency.

Reply:
In addition to consistency, it is also good for retroactively identifying bad app behavior. If `updated_at` does not match `created_at` something that should not have happened did.